docs/unified-test-documentation/dasharo-compatibility/346-hotplug.md

```diff
@@ -0,0 +1,138 @@
+# Dasharo compatibility: Hot-plug
```

Comment:
This test suite contains only test cases dedicated to SATA. So it would be necessary to change its name.

Reply:
Changed